numpy/core/src/umath/loops.c.src

```diff
@@ -786,6 +795,14 @@ NPY_NO_EXPORT void
     }
 }
 
+NPY_NO_EXPORT void
+@TYPE@_isfinite(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    OUTPUT_LOOP {
+        *((@type@ *)op1) = 1;
```

Comment:
So in that future, I guess we would have `np.isna` as well as `np.ismissing` or so and `np.isna` would be something like `np.ismissing | np.isnan`?